frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -75,6 +88,72 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) {
         }
     }
 
+    private void run(
+            ModelServerEndpoint endpoint,
+            FullHttpRequest req,
+            FullHttpResponse rsp,
+            QueryStringDecoder decoder,
+            String method)
+            throws IOException {
```

Comment:
Why IOException?
Should endpoint.goGet() throws some custom Exception, such as MmsPluginException?

Reply:
Is this for future proofing the SDK against the change in the signature?

Reply:
I added the new exception as well.